### PR TITLE
[MWPW-142584 ] Update Review and Modal Block Nala Automation Test script 

### PR DESCRIPTION
**Pull request by skumar09**
PR Includes

- Update the review and modal block test script.
- Reference [Milo PR : 1843]( https://github.com/adobecom/milo/pull/1843)
- Also update `global.setup.js` for local branch run

### CODE REVIEW

File: global.draftsetup.js
Comment (amauch-adobe) [inline global.draftsetup.js:1]: I'm confused what this file is?  I can see it was renamed, but I don't know if it is needed where it is named draftsetup.

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
Everything looked great, I was just confused about the global.setup_1.js rename to global.draftsetup.js.  Does this change or file even need to be committed/merged?

**Review by JackySun9 — APPROVED** [review]

File: global.draftsetup.js
Comment (skumar09) [reply global.draftsetup.js:1]: @amauch-adobe : It is not needed; I have kept it for removing later, ( let me remove it.. )

**Review by amauch-adobe — APPROVED** [review]
Approved.

**Pull request merged**